cnd/src/btsieve/ethereum/mod.rs

```diff
@@ -48,18 +49,23 @@ where
                 }
             }
             GeneratorState::Complete(Err(e)) => return Err(e),
-            GeneratorState::Complete(Ok(infallible)) => match infallible {},
+            GeneratorState::Complete(Ok(_)) => unreachable!(), // Remove this before 0.1 release.
```

Comment:
I would prefer this to be reverted.
The problem with the `unreachable` is that it can actually be hit if the return value of the function ever changes.

With the match on `Infallible`, we get a compile error here if the return value is changed.

Reply:
Cool, good point.  Will remove